src/storage/engines/types.ts

```diff
@@ -490,6 +501,63 @@ export abstract class StorageEngine {
     return sequenceAssignments.map((assignment) => assignment.participantId);
   }
 
+  async saveTags(tags: Tag[], tagType: string) {
+    await this._pushToStorage(`audio/transcriptAndTags/${tagType}`, 'tags', tags);
+  }
+
+  async getTags(tagType: string) {
+    return await this._getFromStorage(`audio/transcriptAndTags/${tagType}`, 'tags', this.studyId);
+  }
```

Comment:
These are all in the `StorageEngine` type, but perhaps would be better in the `CloudStorageEngine`, or even the `FirebaseStorageEngine` since we're relying on firebase to transcribe